AUTHENTICATION.md

```diff
@@ -74,14 +87,12 @@ Console](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-s
 eksctl utils associate-iam-oidc-provider --cluster <cluster-name> --region <region> --approve
 ```
 
-5. Create IAM Role that provider-aws will use
+6. Create IAM Role that provider-aws will use
 
 Set environment variables that will be used in subsequent commands:
 
 ```
-AWS_ACCOUNT_ID=$(aws2 sts get-caller-identity --query "Account" --output text)
 OIDC_PROVIDER=$(aws2 eks describe-cluster --name <cluster-name> --region <region> --query "cluster.identity.oidc.issuer" --output text | sed -e "s/^https:\/\///")
```

Comment:
Why is `aws2` command used in some places (not all)?  I don't get the `aws2` binary (/link? alias?) installed with `awscli` and all of the commands work for me with `aws` in its place.